src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/deployment/lambda_codebase/enable_cross_account_access.py

```diff
@@ -36,6 +36,7 @@ def update_iam(role, s3_bucket, kms_key_arn, role_policies):
 
 
 def lambda_handler(event, _):
+    LOGGER.debug(event)
```

Comment:
Can you remove this debug line, or only expose the data that really needs to be present + explaining where we are at / what we are processing?

Like
```bash
"Setting up cross-account access for ${account_id} in ${regions}"
```

Reply:
I'll just get rid of it. It's a hang over from debugging the step function map.